BankAccount/pkg/store/transactionhistory.go

```diff
@@ -1 +1,73 @@
 package store
+
+import (
+	"bankacc/pkg/entities"
+	"database/sql"
+	"time"
+)
+
+type TransactionHistoryStore interface {
+	Insert(userId int, accountId int, amount float64, action string) (*entities.TransactionHistory, error)
+	GetTransactionsById(id int) (*[]entities.TransactionHistory, error)
+	GetTransactionsByIdFromToDate(id int, fromDate time.Time, toDate time.Time) (*[]entities.TransactionHistory, error)
+}
+
+type TransactionHistoryModel struct {
+	Db *sql.DB
+}
+
+func NewTransactionHistoryStoreModel(db *sql.DB) *TransactionHistoryModel {
+	return &TransactionHistoryModel{
+		Db: db,
+	}
+}
+
+func (store *TransactionHistoryModel) Insert(userId int, accountId int, amount float64, action string) (*entities.TransactionHistory, error) {
+	transaction := entities.TransactionHistory{
+		UserId:    userId,
+		AccountId: accountId,
+		Amount:    amount,
+		Action:    action,
+		CreatedAt: time.Now(),
+	}
+	_, err := store.Db.Exec("INSERT INTO TransactionHistory(user_id, account_id, amount, action, created_at) VALUES(?, ?, ?, ?, ?)", userId, accountId, amount, action, time.Now().Format("2006-01-02T15:04:05"))
```

Comment:
@vpatcev18 no need to Format the string here, just pass `now`.

Reply:
@FilipNikolovski resolved